source/_components/climate.markdown

```diff
@@ -20,6 +20,39 @@ climate:
   platform: demo
 ```
 
+## {% linkable_title Properties %}
+
+Every climate device has a set of properties that control the operation of the climate device. These properties can be set via the frontend or by calling services and are reported in the attributes of the device's state. Note that not all platforms implement all the properties.
+
+We explain each of the properties and the possible choices for them shortly:
+
+* The **action** property specifies the overall operational mode that the device is currently in. Choices are:
+ * Cooling
+ * Heating
+ * Auto (Cool or heat, depending on what's necessary to achieve the target temperature)
+ * Off
+* The **target_temperature** property selects the desired temperature for the device to achieve. The device will take measures to do so depending on what `action` is chosen. While most devices allow only to specify a `target_temperature` and will start heating / cooling when the temperature differs too far from that temperature, some devices allow finer control by specifying an  acceptable temperature corridor:
+  * The **target_temp_high** property selects an upper limit on the acceptable temperature. The device will start cooling (if `cooling` or `auto` are selected as `action`) if that temperature is exceeded.
+  * The **target_temp_low** property selects a lower limit on the acceptable temperature. The device will start heating (if `heating` or `auto` are selected as `action`) if that temperature is exceeded.
```

Comment:
I think that some platforms allow to set a min / max temperature corridor, i.e. "start heating as soon as the temperature drops below `target_temp_low`, start cooling as soon as it exceeds `target_temp_high`". I would have to go through all the platforms though to verify this.

Reply:
Yes, I merely meant that the names should be `target_temperature_high` and `target_temperature_low` so that they are consistent with the "base attribute".

Reply:
Yes, I would certainly do that, as well as renaming any "temperature" or "humidity" without a "current_" or "target_" into the respective one, to avoid all the confusion.

Reply:
There are multiple settings here:
- min and max temperatures = used to avoid freezing or overheating by mistake
- tolerance = most thermostats implement this around target_temperature
- other targets = this is used as separate values for special modes (away, holiday, etc.)

I have to agree that min/max could be reused for away for example and use min for heating and max for cooling, however it really depends on the granularity that most users already use.